### PR TITLE
docs: add quickstart and video image to readme

### PR DESCRIPTION
**Pull request by openoms**
formatted view at:
https://github.com/GaloyMoney/bria/tree/readme-update

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,50 +136,53 @@ Bria enables transaction batching and UTXO management providing the liquidity of
...
+  bria admin bootstrap
   ```
-* create an API key (stored in the .bria folder)
-bria admin bootstrap
-
 * list accounts
   ```
   bria admin list-accounts
   ```
+* create the profile API key (scoped to the account)
```
Comment (bodymindarts) [inline README.md:168]: This creates an account - the profile api key is created as a side effect. You can also explicitly create new profiles and new profile api keys. But an account must be created first.

File: README.md [hunk 2]
```diff
@@ -20,63 +21,113 @@ Bria enables transaction batching and UTXO management providing the liquidity of
...
+  build=unknown-linux-musl
+  # on MacOS use:
+  # build=apple-darwin
+
+  # download
+  wget https://github.com/GaloyMoney/bria/releases/download/${version}/bria-x86_64-${build}-${version}.tar.gz
+
+  # unpack to the ./target/debug directory
+  mkdir -p target/debug
+  tar -xvzf bria-x86_64-${build}-${version}.tar.gz --strip-components 1 -C target/debug
```
[the remark this answers is not in the document]
Comment (openoms) [reply README.md]: ok, using `/usr/local/bin` for the binary

`cargo install bria` will become available once published to crates.io

File: README.md (hunk 1 above)
Comment (openoms) [reply README.md]: corrected

File: README.md [hunk 3]
```diff
@@ -85,50 +136,53 @@ Bria enables transaction batching and UTXO management providing the liquidity of
...
+  ```
+* create a single-sig wallet for the default bria account
+  ```
+  bria create-wallet -n default descriptors -d "wpkh([6f2fa1b2/84'/0'/0']tpubDDDDGYiFda8HfJRc2AHFJDxVzzEtBPrKsbh35EaW2UGd5qfzrF2G87ewAgeeRyHEz4iB3kvhAYW1sH6dpLepTkFUzAktumBN8AXeXWE9nd1/0/*)#l6n08zmr" \
+      -c "wpkh([6f2fa1b2/84'/0'/0']tpubDDDDGYiFda8HfJRc2AHFJDxVzzEtBPrKsbh35EaW2UGd5qfzrF2G87ewAgeeRyHEz4iB3kvhAYW1sH6dpLepTkFUzAktumBN8AXeXWE9nd1/1/*)#wwkw6htm"
+  ```
 
-For testing see the commands in the [helpers.bash file](tests/e2e/helpers.bash)
-
+* For testing further see the commands in the [helpers.bash file](tests/e2e/helpers.bash)
```
[the remark this answers is not in the document]
Comment (openoms) [reply README.md]: referring to to the command `--help` -s now

**Pull request merged**